lib/traject/bibtex_config.rb

```diff
@@ -38,7 +38,7 @@
 to_field 'issue_ssm', extract_bibtex_field(:issue)
 to_field 'pages_ssm', extract_bibtex_field(:pages)
 to_field 'doi_ssim', extract_bibtex_field(:doi)
-to_field 'general_notes_ssim', extract_bibtex_field(:annote, split: '|', trim: true)
+to_field 'general_notes_ssim', extract_bibtex_field(:annote, split: '\\textbar', trim: true)
```

Comment:
If you want, you can make this change in the preprocessing code so do a `gsub('\\textbar', '|')` here -- https://github.com/sul-dlss/exhibits/blob/master/app/models/bibliography.rb#L44

Reply:
Thanks for the tip! That worked perfectly.